test/e2e/operator.go

```diff
@@ -378,34 +385,127 @@ var _ = Describe("ARO Operator - Azure Subnet Reconciler", func() {
 	})
 })
 
-var _ = Describe("ARO Operator - MUO Deployment", func() {
-	ctx := context.Background()
+func getMUOPods(ctx context.Context) (*corev1.PodList, error) {
+	listOptions := metav1.ListOptions{
+		LabelSelector: "name=managed-upgrade-operator",
+	}
+	pods, err := clients.Kubernetes.CoreV1().Pods(managedUpgradeOperatorNamespace).List(ctx, listOptions)
+	if err != nil {
+		return nil, err
+	}
 
-	It("must be deployed by default with FIPS crypto mandated", func() {
-		muoIsDeployed := func() (bool, error) {
-			By("getting MUO pods")
-			pods, err := clients.Kubernetes.CoreV1().Pods("openshift-managed-upgrade-operator").List(ctx, metav1.ListOptions{
-				LabelSelector: "name=managed-upgrade-operator",
-			})
-			if err != nil {
-				return false, err
-			}
-			if len(pods.Items) != 1 {
-				return false, fmt.Errorf("%d managed-upgrade-operator pods found", len(pods.Items))
-			}
+	if len(pods.Items) != 1 {
+		return nil, fmt.Errorf("%d managed-upgrade-operator pods found", len(pods.Items))
+	}
+	return pods, nil
+}
 
-			By("getting logs from MUO")
-			b, err := clients.Kubernetes.CoreV1().Pods("openshift-managed-upgrade-operator").GetLogs(pods.Items[0].Name, &corev1.PodLogOptions{}).DoRaw(ctx)
-			if err != nil {
-				return false, err
-			}
+func getMUODeployment(ctx context.Context) *appsv1.Deployment {
+	deploymentList, err := clients.Kubernetes.
+		AppsV1().
+		Deployments(managedUpgradeOperatorNamespace).
+		List(ctx, metav1.ListOptions{})
+
```

Comment:
you should use get here, you know the name of the deployment you want so you might as well just try to retrieve it directly instead of listing and checking the length. This test would break if we are to deploy extra deployments in that namespace.

Reply:
Yes, you're right. Done. Thanks! 😃